cognite_toolkit/_cdf_tk/pull.py

```diff
@@ -464,7 +470,8 @@ def pull_command(
         )
 
     if not dry_run:
-        source_file.write_text(new_content)
+        with source_file.open(mode="w", encoding=ENCODING, newline=NEWLINE) as f:
+            f.write(new_content)
```

Comment:
No issues on windows skipping carriage return as part of newline?

Reply:
No issues. Already tested this with pygen and neat which also write files to disk.